VitalTrackApp/App.js

```diff
@@ -31,6 +32,7 @@ function AppNavigator() {
       </Stack.Screen>
       <Stack.Screen name="Settings" component={SettingsScreen} />
       <Stack.Screen name="Entries" component={EntriesScreen} />
+       <Stack.Screen name="Calendar" component={CalendarScreen} options={{ headerShown: false }} />
```

Comment:
why are we hiding the header for the calendar but not everything else?

Reply:
I'll put it back i was having issues with the CSS for a calendar so I omitted  out to try to fit the calendar better

Reply:
you can either take it out or omit the header for every other component

Reply:
I hid the header for all screens